cmd/fleet/cron.go

```diff
@@ -41,54 +42,24 @@ func cronVulnerabilities(
 	logger = kitlog.With(logger, "cron", lockKeyVulnerabilities)
 
 	if config.CurrentInstanceChecks == "no" || config.CurrentInstanceChecks == "0" {
```

Comment:
Not sure if this was intended - but we won't be able to detect changes to these config options while running.

Reply:
Oh this is a config option not an app config option.